ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -503,35 +444,24 @@ export const AdminInstitutionEdit = fp.flow(withNavigation, withRouter)(class ex
   }
 
   get fieldsEditedEditInstitution() {
-    const {institution, institutionToEdit} = this.state;
-    return institution !== institutionToEdit;
+    const {institution, institutionBeforeEdits} = this.state;
+    return institution !== institutionBeforeEdits;
   }
 
   hasInvalidFields() {
     const {institution} = this.state;
-    let emailValid = true;
     if (!institution.tierConfigs) {
       // It is not expected for a tier requirement to be empty
       return true;
     }
-    const rtConfig: InstitutionTierConfig = getRegisteredTierConfig(institution);
-    if (rtConfig.membershipRequirement !== InstitutionMembershipRequirement.NOACCESS) {
-      emailValid = rtConfig.membershipRequirement === InstitutionMembershipRequirement.DOMAINS ?
-          rtConfig.emailDomains !== undefined : rtConfig.emailAddresses !== undefined;
-    }
-    return !emailValid || !institution.displayName || !institution.organizationTypeEnum ||
```

Comment:
`displayName` was already covered by `validate.js` and now `organizationTypeOtherText` is as well.